src/pages/Landing/index.tsx

```diff
@@ -56,6 +56,34 @@ export default () => {
   const columnHeaders = ['Name', 'Age', 'Is Cool'];
 
   if (isLoading) return <Loader />;
+  function handleDelete(customerId: any): void {
+    const deleteCustomer = () => {
+      const customers = customersGetter();
+      const index = customers.findIndex((customer) => customer.name === customerId);
+
+
+      if (index !== -1) {
+        const updatedCustomers = [...customers];
+        updatedCustomers.splice(index, 1);
+        customersSetter(updatedCustomers);
+      }
+    };
+
+    try {
+      deleteCustomer();
+      queryClient.invalidateQueries('customers');
+    } catch (error) {
+      console.error('An error occurred while deleting the customer:', error);
+
+    }
+  }
+  function generateActionItems(name, handleEdit, handleDelete) {
```

Comment:
you can call handleDelete() directly within this function, no need to pass it in.